src/types/actions.ts

```diff
@@ -168,3 +168,10 @@ export enum DecisionMethod {
   Permissions = 'Permissions',
   Reputation = 'Reputation',
 }
+
+export enum TokenStatus {
+  Added = 'Added',
+  Removed = 'Removed',
+  Unaffected = 'Unaffected',
+  NotEditable = 'NotEditable',
+}
```

Comment:
Considering this status only exists in the context of ActionSidebar form, should it be moved closer to that component?